docker/base/Dockerfile

```diff
@@ -24,6 +24,14 @@
 ARG bootstrap_version=9
 ARG image="vitess/bootstrap:${bootstrap_version}-mysql57"
 
+FROM node:16-bullseye-slim as node
+
+COPY web/vtadmin /vt/web/vtadmin
+
+# install/build/clean web dependencies
+RUN npm --prefix /vt/web/vtadmin ci && \
+    npm run --prefix /vt/web/vtadmin build
+
```

Comment:
If we are adding this to the bootstrap image, then we don't need to add it here.
Instead, we need to first build and push bootstrap images with a new version number (10), and increment the version number in all the Dockerfiles that use that as the base.

Reply:
based on offline discussion, I now see that we are actually _building_ vtadmin in this step. So it doesn't belong in the bootstrap image at all.
bootstrap image should contain static dependencies (like node and npm). Builds need to happen in base and lite.
I was confused because the changes were also made to the bootstrap image.
One way forward is to revert changes to bootstrap image and make them in all the lite images. Then there is no need for a new bootstrap version either.
I'll leave it up to you, @ajm188 and @GuptaManan100 to decide whether node/npm need to be installed in the bootstrap image at all.